sklearn/feature_selection/_sequential.py

```diff
@@ -201,11 +199,23 @@ def fit(self, X, y=None):
             Target values. This parameter may be ignored for
             unsupervised learning.
 
+        **params : dict, default=None
+            Parameters to be passed to the cross_val_score function.
```

Comment:
Ah right. In that case, perhaps the following to allow users to click in and see what is being done underneath the hood?

```suggestion
            Parameters to be passed to the :func:`cross_val_score` function.
```